docs/05-reference/examples.mdx

```diff
@@ -13,4 +13,5 @@ into your project.
 | [Justcast.it](https://github.com/victorges/justcast.it)                       | This web app provides code for adding "1-click starting a livestream" so that you can create a stream using the Livepeer Studio API.                                                                                                                                                                                                                                                                                                                   |
 | [Livepeer Web API Proxy](https://github.com/victorges/livepeer-web-api-proxy) | This Proxy extends the ability to limit access to the Livepeer Studio API for the web so that you can implement your own authentication method (e.g., if security or billing are of concern). You will be able to allow for creating streams directly from the browser without exposing access to an entire account via your API key. You can maintain, i.e. iterate and update, on the live deployment leveraging [fly.io](https://fly.io/) features. |
 | [Livepeer NFT Gate](https://github.com/livepeer/livepeer-nft-gate-example)    | This code provides steps to build a Livepeer NFT Gate app so that you can validate that, from a specific contract, a viewer owns an NFT in order to view your livestream.                                                                                                                                                                                                                                                                              |
-| [Livepeer Studio Sample App](https://github.com/livepeer/studio-sample-app)           | This sample application demonstrates the capabilities of what to expect when building with Livepeer Studio. When building a project with Livepeer Studio, there are examples provided for features such as On Demand or the Video Player. These examples provide a more in-depth approach using the API or plug-and-play with the SDK.                                                                                                                                                                                     |
\ No newline at end of file
+| [Livepeer Studio Sample App](https://github.com/livepeer/studio-sample-app)   | This sample application demonstrates the capabilities of what to expect when building with Livepeer Studio. When building a project with Livepeer Studio, there are examples provided for features such as On Demand or the Video Player. These examples provide a more in-depth approach using the API or plug-and-play with the SDK.                                                                                                                 |
```

Comment:
Let's update this copy to the following:

This example demonstrates how to add live or on demand video experiences to your application using the Livepeer API and [Livepeer.js](https://livepeerjs.org).

Reply:
Let's also rename "Livepeer Studio Sample App" to "Livepeer API usage examples"